### ISSUE TITLE
Accessibility Testing for MHV-Horizon, MHV on VA.gov, Notification email update critical alerts

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#122443]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@steveg-IAT

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

- Colorblindly and Wave checks passed.

**Known issue**: 
- Element focus indicator (#FACE02) on Yellow alert background (#FAF3D1) does not meet non-text contrast requirements of 3:1 - [SC 1.4.11](https://w3.org/TR/WCAG22#non-text-contrast) && [SC 2.4.13](https://w3.org/TR/WCAG22#focus-appearance) - current contrast ratio: 1.35:1.

### Who completed the axe scans?

@steveg-IAT

### axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

- Components **passed** Axe (and Lighthouse) scans.

### How did axe testing go?

- No issues with the components.

### Who completed the content zoom and reflow test?

@steveg-IAT

### Content zoom and reflow checks

- [x] (Web only) All page elements are readable and usable at 200% zoom
- [x] (Web only) All page elements are readable and usable at 300% zoom
- [x] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

- Components passed zoom/reflow tests.

### Who completed the keyboard navigation test?

@steveg-IAT

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

- Keyboard nav checks passed.

### Do you have any other results to share?

**MyVA && MHV:** 
- Clicking on "[Go to profile to update your contact email](https://staging.va.gov/profile/contact-information#contact-email-address)" targets `profile/contact-information#contact-email-address`, but the target id on the profile page is `id="email-address"`. 
  - Figma indicates that the alert component should also be shown on the profile page, between the 'Email address' heading and the 'contact email' card component - there is no alert present. 
    - Waiting on: [Node 22 CI check](https://github.com/department-of-veterans-affairs/vets-website/pull/39261) to verify this flow and components.
    - _CONFIRMING_ this alert is present in the correct position on the profile page.
  - [new ticket #122590](https://github.com/department-of-veterans-affairs/va.gov-team/issues/122590)
- Clicking the "Confirm" button adds a new alert component to the page above the warning/critical action alert, but does not focus on the new alert or otherwise indicate its presence non-visually.  [SC 4.1.3](https://w3.org/TR/WCAG22#status-messages)  
  - New elements need `role="alert"` 
  - [new ticket #122600](https://github.com/department-of-veterans-affairs/va.gov-team/issues/122600)

**Comment by steveg-IAT** [comment]
**General Comments**
- Having intermittent issues with ID.me and 500 error (they are apparently rolling out a new release). 
- Only getting Scenario 1 "Confirm" and "couldn't confirm email" flows with [users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-mhv-cartographers.md#email-alerts-test-users) listed in scenario 1 and 2.
- As noted in the additional/other results section, waiting on PRs to clear before I can check the profile page.
- Will add ticket links asap.

**Note**
- Add JasonDay back to ticket once completed.
